rollupConfigFactory.js

```diff
@@ -0,0 +1,214 @@
+import fs from 'fs';
```

Comment:
the idea of moving this function to this common file is good 👍 
one details is I think for consistency would stick to "kebab" casing, `rollup-config-factory.js`?